packages/firestore/src/lite-api/aggregate_types.ts

```diff
@@ -15,24 +15,45 @@
  * limitations under the License.
  */
 
+import { AggregateType } from '../core/aggregate';
+import { ObjectValue } from '../model/object_value';
+import { FieldPath as InternalFieldPath } from '../model/path';
+
 import { Query } from './reference';
+import { AbstractUserDataWriter } from './user_data_writer';
+
+export { AggregateType };
 
 /**
  * Represents an aggregation that can be performed by Firestore.
  */
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export class AggregateField<T> {
   /** A type string to uniquely identify instances of this class. */
-  type = 'AggregateField';
+  readonly type = 'AggregateField';
+
+  /**
+   * Create a new AggregateField<T>
+   * @param _aggregateType Specifies the type of aggregation operation to perform.
+   * @param _internalFieldPath Optionally specifies the field that is aggregated.
+   * @internal
+   */
+  constructor(
+    // TODO (sum/avg) make aggregateType public when the feature is supported
+    readonly _aggregateType: AggregateType = 'count',
```

Comment:
does it still make sense to have `= 'count'` ?

Reply:
Previously the default constructor was public. It's unlikely, but someone could have instantiating this class without any params, so I wanted to support the same. Defaulting to 'count' was equivalent.

Reply:
I see. Thanks